Tutorials/Make_Your_Own_Intake_Datastore.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>Maybe don't show <code>'Builder'</code> - this is a class taken from another package that <code>BaseBuilder</code> inherits from. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/440/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
ok done